app/javascript/src/QuickStarts/templates/createMetric.yaml

```diff
@@ -0,0 +1,106 @@
+
+    _Hits_ is the built-in metric that exists for each API. It tracks the number of calls made to your API. To capture usage of your API apart from _Hits_, define a metric that reports the usage in different units. A unit should be quantifiable and apply a meaning for your business goals such as megabytes (MB), CPU time, or the number of elements returned by the API. Metrics other than _Hits_, such as CPU time or MB are not provided by default. Obtain these metrics by using an endpoint called by an external service configured by the user.
+
+    You can define metrics for products and backends.
+  tasks:
+    - title: Add metrics to backend
+      description: |-
+        Adding a metric specifies a usage unit that you want to capture for all calls to your API. An application plan provides the ability to set a limit for each metric that you add to a product or backend.
+
+        ### Prerequisites
+
+        - A backend. To create one, see Creating a backend.
+        
+        ### Procedure
+
+        1. Navigate to <strong>[Your\_backend\_name] > Methods and Metrics</strong>.
+        1. Click <strong>New metric</strong>.
+        1. On the _New metric_ page, fill in the details to create the metric.
+        
+          Important fields:
+              - _Friendly name_: A short description of the metric. This name is displayed in different sections of the 3scale Admin Portal.
+              - _System name_: The name of the metric in your API to use to report the usage through the 3scale Service Management API. The system name must be unique.
+              
+              > <strong>Tip:</strong> Be careful with changing the system name of the metrics or deleting them. These changes can break your already deployed 3scale integration if there are mapping rules pointing to the previous system name of the metric.
+              
+              - _Unit_: Use a singular noun, for example, _hit_. The singular will become plural in the analytics charts.
+        1. Click <strong>Create metric</strong>
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you create a unique system name for the metric?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+    - title: Add metrics to product
+      description: |-
+        The procedure for adding a metric to a product is similar to adding a metric to a backend.
+
+        ### Prerequisites
+
+        - A Product. To create one, see Creating a product.
+        - An application plan. To create one, see Create an application plan.
```

Comment:
see 'Create an application plan' is a link, which link to 'createApplicationPlan.yaml'